developers/weaviate/config-refs/schema/index.md

```diff
@@ -132,7 +132,17 @@ The following are not allowed:
 * Any map type is forbidden, unless it clearly matches one of the two supported types `phoneNumber` or `geoCoordinates`.
 * Any array type is forbidden, unless it is clearly a reference-type. In this case, Weaviate needs to resolve the beacon and see what collection the resolved beacon is from, since it needs the collection name to be able to alter the schema.
 
-### Multiple vector embeddings (named vectors)
+### Single vector collections
+
+If you don't explicitly define a [named vector](#named-vectors) in your collection definition, Weaviate automatically creates what's known as a *single vector* collection. These vectors are stored internally under the named vector `default` (which is a reserved vector name).
```

Comment:
Is this true? I think this is the plan, but I was not sure if this has been implemented.

Reply:
Some further context: https://weaviate-org.slack.com/archives/C05M1Q125UY/p1747149106303419?thread_ts=1747144188.375159&cid=C05M1Q125UY